service/src/test/java/com/codeforcommunity/JooqMock.java

```diff
@@ -16,10 +16,7 @@
 import org.jooq.Table;
```

Comment:
Hmm, this class looks like it's the old version still. Have you pushed the new version?

Reply:
Looks like it's been updated.